frontend/models/Text2Text.js

```diff
@@ -20,10 +20,12 @@ import { BaseRecord, BaseSearchQuery, BaseSearchResults } from "./Common";
 
 class Text2TextRecord extends BaseRecord {
   text;
+  recordViewSettings;
 
-  constructor({ text, ...superData }) {
+  constructor({ text, recordViewSettings, ...superData }) {
     super(superData);
     this.text = text;
+    this.recordViewSettings = recordViewSettings;
```

Comment:
we can set here the default value for recordViewSettings.

Better naming could be `viewSettings` or `viewState`  since is already defined in a record